api/include/opentelemetry/plugin/tracer.h

```diff
@@ -50,9 +50,13 @@ class Span final : public trace::Span
 
   trace::Tracer &tracer() const noexcept override { return *tracer_; }
 
+  context::Token *GetToken() const noexcept override { return nullptr; }
+
+  void SetToken(context::Token *token) noexcept override {}
```

Comment:
Raw pointer in this API looks suspicious and most likely violates C++ Core Guidelines.

Reply:
Changed to unique_ptr.